src/transformers/modeling_lxmert.py

```diff
@@ -0,0 +1,1348 @@
+        """ Initialize the weights """
+        if isinstance(module, (nn.Linear, nn.Embedding)):
+            # Slightly different from the TF version which uses truncated_normal for initialization
+            # cf https://github.com/pytorch/pytorch/pull/5617
+            module.weight.data.normal_(mean=0.0, std=self.config.initializer_range)
+        elif isinstance(module, LxmertLayerNorm):
+            module.bias.data.zero_()
+            module.weight.data.fill_(1.0)
+        if isinstance(module, nn.Linear) and module.bias is not None:
+            module.bias.data.zero_()
+
+
+LXMERT_START_DOCSTRING = r"""
+    The LXMERT model was proposed in `LXMERT: Learning Cross-Modality Encoder Representations from Transformers`
+    by Hao Tan and Mohit Bansal. It's a vision and language transformer model,
+    pre-trained on a variety of multi-modal datasets comprising of GQA, VQAv2.0, MCSCOCO captions, and Visual genome,
+    using a combination of masked language modeling, region of interest feature regression,
+    cross entropy loss for question answering attribute prediction, and object tag predicition.
+
+    This model is a PyTorch `torch.nn.Module <https://pytorch.org/docs/stable/nn.html#torch.nn.Module>`_ sub-class.
+    Use it as a regular PyTorch Module and refer to the PyTorch documentation for all matter related to general
+    usage and behavior.
+
+    Parameters:
+        config (:class:`~transformers.LxmertConfig`): Model configuration class with all the parameters of the model.
+            Initializing with a config file does not load the weights associated with the model, only the configuration.
+            Check out the :meth:`~transformers.PreTrainedModel.from_pretrained` method to load the model weights.
+"""
+
+LXMERT_INPUTS_DOCSTRING = r"""
+
+    Args:
+        input_ids (:obj:`torch.LongTensor` of shape :obj:`{0}`):
+            Indices of input sequence tokens in the vocabulary.
+
+            Indices can be obtained using :class:`transformers.LxmertTokenizer`.
+            See :func:`transformers.PreTrainedTokenizer.encode` and
+            :func:`transformers.PreTrainedTokenizer.__call__` for details.
+
+            `What are input IDs? <../glossary.html#input-ids>`__
+        visual_feats: (:obj:`torch.FloatTensor` of shape :obj:՝(batch_size, num_visual_features, visual_feat_dim)՝):
+            This input represents visual features. They ROI pooled object features from bounding boxes using a faster-RCNN model)
+            These are currently not provided by the transformers library
+        visual_pos: (:obj:`torch.FloatTensor` of shape :obj:՝(batch_size, num_visual_features, visual_pos_dim)՝):
+            This input represents spacial features corresponding to their relative (via index) visual features.
+            The pre-trained lxmert model expects these spacial features to be normalized bounding boxes on a scale of 0~1.
+            These are currently not provided by the transformers library
```

Comment:
Here we should add an URL or link to where people can find the missing features (or a link to the future example showing how to generate them?)